src/components/breadcrumb/breadcrumb.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { BreadcrumbItem } from "./breadcrumb-item";
+import { BreadcrumbArrow } from "./breadcrumb-arrow";
+
+export interface BreadcrumbProps {
+    children: React.ReactNode;
+}
+
+const Breadcrumb = ({ children }: BreadcrumbProps) => {
```

Comment:
The wrapper should be named in plural, thats a more common convention

Reply:
renamed it to breadcrumbs ✅